src/components/application_manager/include/application_manager/application_impl.h

```diff
@@ -180,6 +196,13 @@ class ApplicationImpl : public virtual InitialApplicationDataImpl,
   DataAccessor<VehicleInfoSubscriptions> SubscribedIVI() const OVERRIDE;
   inline bool IsRegistered() const OVERRIDE;
 
+  bool SubscribeToInteriorVehicleData(smart_objects::SmartObject module);
+  bool IsSubscribedToInteriorVehicleData(smart_objects::SmartObject module);
+  bool UnsubscribeFromInteriorVehicleData(smart_objects::SmartObject module);
+
+#ifdef SDL_REMOTE_CONTROL
```

Comment:
@LuxoftAKutsan, @okozlovlux, perhaps, it would be better to move all members related to `SDL RC` functionality to one `#ifdef`, or at least  reduce their amount everywhere